doc/website/concepts/qos-policies.md

```diff
@@ -0,0 +1,33 @@
+# QoS policies
+
+Unlike other middlewares, e.g. [DDS used in ROS 2](https://docs.ros.org/en/galactic/Concepts/About-Quality-of-Service-Settings.html),
+iceoryx keeps things simple and offers basic Quality of Service (QoS) settings. The QoS
+settings are called options and can be used to optimize and tailor the communication.
+
+## `PublisherOptions` and `SubscriberOptions`
```

Comment:
Do we need the same for `ClientOptions` and `ServerOptions`?

Reply:
Good point, updated.